### ISSUE TITLE
Allow user to change import settings after hub has been created

### ISSUE THREAD

**Issue by whyboris** [whyboris/Video-Hub-App#468]
This is really easy, but will take some time to make the UI look pretty.

I'll work on this feature after version 3.0.0 is released 👍

**Comment by cal2195** [comment]
Will it retroactively regenerate all previously imported videos? 🤔

**Comment by whyboris** [comment]
I don't think it should. It will affect only new videos (I'll need to check it doesn't cause problems with different image heights). It will probably add all the missing clips if the user enables clips. 

I don't see this as an important feature, but if it works well without much effort I'll add it.

If the user wants to re-generate all the images they could delete the contents of the "filmstrips" folders 🤷

**Comment by orkomlosh** [comment]
@whyboris This feature will be only after 3.0.0 is released right?
just wondered about this, import settings are actually the wizard settings when creating new hub right?
so maybe the easy way is to use this screen. 
we can use the history list, to add a new "edit" button to each line which will load the settings of the hub on the creation form, and buttons will changed to save and/or save and open.
also can edit hub from file (like open from file),
what do you think?

**Comment by whyboris** [comment]
I'm eager to release `3.0.0` which is why I'd probably push this change until later. This change doesn't interfere with anything, so it's not like I need to get it out before `3.0.0` is finalized 🤔 

I think I have a vague picture of what you're describing: letting users edit the settings by clicking a newly-added `Edit` button next to items on the "Recently Opened" list.

_My initial thoughts about how it would look:_
The "Statistics" tab in `2.X.X` is now "Current Hub" tab in `3.X.X` and it still shows the statistics about all files and the extraction settings. I was imagining an "Edit" mode there which would change the numbers in those columns to dropdowns & number-input fields. It would be very clear about what each one does.

I'm thinking conceptually it makes sense to edit the hub settings in the place where you can add/remove folders to/from the hub. The UI is almost ready -- just a mater of toggling input boxes.

While we _could_ let users edit hub settings from the wizard-like interface, I think it may be less elegant than the "Current Hub" location I had in mind 🤔 

I'm open to a discussion on this. Sorry I didn't make it clear what I had in mind initially.

**Comment by orkomlosh** [comment]
i'm with you it sound better, this is full description of what need to be done. let's see who will get to it first 😄

**Comment by whyboris** [comment]
I'm going to push this back for a while -- I am too eager to release `3.0.0` first, and then to work on the _server_ aspect of the app 😎

**Comment by adamwinn** [comment]
I'm a big fan of this. If I recreate my hub incorrectly, I hate to have to scrap it and start over.  For example, I'd like to be able to change it from 5 thumbnails to 10 thumbnails without starting all over.

**Comment by whyboris** [comment]
Thanks for chiming in @adamwinn -- now that version `3.0.0` is released, we can get started on more features 😁 

From the code perspective this feature is rather straight forward -- just will take some time to update the UI (and make it look nice) and take care to delete all thumbnails if thumbnail settings change 👌
